### PR TITLE
MariaDB 10.6 is not supported

### PR DESCRIPTION
**Pull request by kesselb**

### CODE REVIEW

**Review by skjnldsv — APPROVED** [review]

File: admin_manual/installation/system_requirements.rst [hunk 1]
```diff
@@ -21,7 +21,7 @@ For best performance, stability and functionality we have documented some recomm
 |                  | - openSUSE Leap 42.1+                                                 |
 |                  | - CentOS 8                                                            |
 +------------------+-----------------------------------------------------------------------+
-| Database         | - **MySQL 8.0+ or MariaDB 10.2+** (recommended)                       |
+| Database         | - **MySQL 8.0+ or MariaDB 10.2/10.3/10.4/10.5 ** (recommended)        |
```
Comment (rugk) [inline admin_manual/installation/system_requirements.rst:24]:
```suggestion
| Database         | - **MySQL 8.0+ or MariaDB 10.2/10.3/10.4/10.5** (recommended)        |
```

You have a syntax error here…

**Review by rugk — CHANGES_REQUESTED** [review]

File: admin_manual/installation/system_requirements.rst (hunk 1 above)
Comment (kesselb) [reply admin_manual/installation/system_requirements.rst]: Thanks :+1:

**Comment by My1** [comment]
I went ahead and did the manual backport, as I didnt see any other PR for it.

**Pull request merged**